### PR TITLE
Fixed icon sources.

### PR DESCRIPTION
**Pull request by Bingies24**
Windows - Completely redone.
MacOS - Completely redone. Uses accurately shaped icons and has @2x's different square as well.
Linux - Completely redone.

### CODE REVIEW

**Comment by Schneegans** [comment]
Hi there! Thanks a lot. However, I would like to know a bit more about your reasoning here:
* 128px Windows size? I guess we can add it; however it's optional and as it contains the same blossom, it really doesn't matter. If Windows downscales it from 256x256 it will be exactly the same result as it's a factor of two.
* I think we do not need a different background for Windows and Linux. We can use the same SVG but use a different margin.
* What's the difference between the macOS icons? I do not see any difference.
* What did change in `bg_tiny_square.svg`?
* All your SVGs contain way too much useless defs. They are all more than 200kB+ where less than 10kB should be sufficient. Just have a look at the diff, you almost added 20.000 lines!

Overall, I think the only real "fix" here is that the Windows icon now covers the entire icon, which is indeed a valid improvement. Or do you see this differently?

**Comment by Bingies24** [comment]
Shoot, I didn't export them as optimized, that's why they're so big.
MacOS doesn't use a rounded square but a specific signature shape. I changed it to use that shape instead like it was pre-source split. `bg_tiny_square.svg` also uses the same square.
I just wanted Windows to not have to use margins for the base since it looked better in my opinion. What do you have in mind instead?

**Comment by Bingies24** [comment]
128x128 because it might be better to make a rendered size for it rather than trust Windows to scale it down right, but that's just me. It's probably fine as is but it should be checked in Windows itself to see if it's better or worse than making a direct render for it.

**Comment by Schneegans** [comment]
Yeah, I think the Windows fixes are good. But still: What's the difference between the macOS icons?

File: icons/create.sh [hunk 1]
```diff
@@ -232,19 +232,21 @@ WIN_TMP_DIR=$TMP_DIR/win
...
+composite_svgs_and_save_as_png source/bg_circle_windows.svg source/blossom_large.svg "$WIN_TMP_DIR/128.png"  0 17 128
+composite_svgs_and_save_as_png source/bg_circle_windows.svg source/blossom_large.svg "$WIN_TMP_DIR/256.png" 0 17 256
 
 convert \
   "$WIN_TMP_DIR/16.png" \
   "$WIN_TMP_DIR/32.png" \
   "$WIN_TMP_DIR/48.png" \
   "$WIN_TMP_DIR/64.png" \
   "$WIN_TMP_DIR/96.png" \
+  "$WIN_TMP_DIR/128.png \
```
Comment (Schneegans) [inline icons/create.sh:249]:
```suggestion
  "$WIN_TMP_DIR/128.png" \
```

**Comment by Schneegans** [comment]
As discussed on Discord, I integrated most of your changes here: https://github.com/kando-menu/design/commit/fbe783f7613cf9ae894ae7651e5ee85c9e6a003b. Thanks again for the fixes!

**Pull request closed without merging**